cmake/SetupLicenseInfo.cmake

```diff
@@ -0,0 +1,74 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(_LIST_OF_3PLS
+  Blaze
+  Boost
+  Brigand
+  Catch2
+  Gsl
+  Hdf5
+  Libbacktrace
```

Comment:
Should jemalloc be here?  I think that's the only missing link dependency we have listed (other than charm).

Reply:
We don't always link it, but I'll add it anyway. That way we don't have to worry about it when distributing executables :)